src/sdk/base/publication.js

```diff
@@ -15,7 +15,14 @@ import {EventDispatcher} from '../base/event.js';
  */
 export class AudioPublicationSettings {
   // eslint-disable-next-line require-jsdoc
-  constructor(codec) {
+  constructor(id, codec) {
+    /**
+     * @member {?Owt.Base.AudioCodecParameters} id
+     * @instance
+     * @classDesc Identifer for this set of publication settings.
+     * @memberof Owt.Base.AudioPublicationSettings
+     */
+    this.id=id,
```

Comment:
Why we need an ID here?

Reply:
The identifier for which track to subscribe for subscription, contains mid.